\chapter{Introduction}

\section{Using genetics to understand complex traits and disease}

The overarching goal of human genetics is to understand how genetic variation influences phenotypes-- including complex traits and diseases-- within our species.  One approach to connect genotypic to phenotypic variation is to compare humans with non-human primates. This approach is particularly compelling given that human share over 99\% of their DNA sequence with chimpanzees (REF). Efforts have been made to understand the 1\%, that is to say the impact of these approximately 30 million single nucleotide polymorphisms (SNPs) differences (REF). Importantly, the majority of these SNPs are in the non-protein coding regions of the genome (REF).  Overall this high degree of sequence similarity led Mary Claire King and Greg Wilson to hypothesize that it is the way these DNA sequences are regulated that leads to phenotypic differences (REF King and  Wilson). Consequently, comparisons between humans and non-human primates can help to determine how these DNA sequences function and to reveal the underlying mechanisms that act on or as a result of sequence variation. This desire for for mechanistic understanding has inspired a number of studies to compare gene regulation levels (REF), epigenetic marks (REF), protein levels, and various other molecular phenotypes. These efforts have resulted in large comparative genomic catalogs of similarities and differences in gene regulation between humans and other primates. These catalogs have great potential to help us better understand the evolutionary processes that led to adaptations in humans [3, 4, 6, 13, 16-26], establish informed models of the relative importance of changes in different molecular mechanisms to regulatory evolution [27, 28], and identify molecular pathways that may be functionally important in the context of complex diseases (REF Gallego Romero review). However, the genetic variants that drive phenotypic diversity often have small effects, which are difficult to detect even in well-powered studies (REF). It is therefore essential to design comparative studies that allow us to isolate the variables of interest while minimizing the effects of unwanted biological and technical differences. Yet, all too often, various aspects of study design are overlooked, to the detriment of the field. In extreme cases, poor study design leads to erroneous inference and incorrect interpretations (REF Mouse Encode; Gilad and Man 2015). The majority of the time, a poor study design limits the accuracy of the study and by extension, the biological insight that can be drawn from it (REF Pai et al 2011). Compounding the issue is that study design considerations are usually not explicitly discussed in comparative genomic papers. The majority of my work has focused on primate comparative genomics - an area which presents many opportunities to discuss how an effective study design can affect the results of a study and aid in its interpretation. In this Introduction, I focus on two principles critical to identifying robust biological differences between species: minimizing confounders and careful sample collection. However, the principles of study design that I will discuss extend beyond primate comparative genomics to any study that makes comparisons between groups, including case-control studies in humans.\section{Study design challenges in comparative primate genomics}

\textit{Confounders and other potential biases.} As the technology used in genomic studies has progressed, so too has our understanding of the widespread nature and impact of confounders and other potential biases. Furthermore, using multiple species in functional genomic studies has increased the difficulty of minimizing these confounders. For example, early comparative studies using gene expression and DNA methylation microarrays often did not account for the attenuation of hybridization caused by sequence mismatches, which differ between species [6, 17, 18, 31]. More recently, common confounders of sequence-based comparative studies include individual sampling schemes that are unbalanced across species, and sample processing steps that are segregated by species [2, 10, 21, 32-34]. Systemic differences inherent to the samples, such as differences in material quality between species [2, 35], also remain a concern. Similarly, a primate comparative framework brings forth analytical challenges. For example, analyses that do not use orthologous sequences or effective normalization procedures can result in bias [10, 21, 35]. Yet, most comparative genomic studies of humans and non-human primates that we are aware of, including previous studies from our own group [10, 21, 35], suffer from one or more of these weaknesses and caveats. Very few people would disagree that it is important to use good study design. However, the number of potential confounders are vast, including sex, date of death, age, RNA concentration, RIN score, RNA extraction date, library concentration, index sequence, sequencing pool, sequencing location, sequencing lane, total sequencing reads. Therefore, it can be difficult to detect confounding factors and bias introduced during sample processing or data analysis. Unfortunately, these factors are often neither accounted for nor discussed. Sometimes, this can lead to erroneous conclusions, which could have major implications for biological research. For example, a recent paper claimed that global gene expression levels were driven by species rather than tissue type (Yue et al. 2014). Upon re-analysis, it was uncovered that the human and mouse samples used in the study were sequenced in different batches (Gilad and Mizrahi-Man). With this study design, the biological variable of interest (species) is confounded with the technical variable of sequence batch. Therefore, it is unknown to what extent the technical variable drove the biological results reported in the original paper. Sorting out which results were driven by biological, rather than technical differences, is often led to the reader. Such a task can be quite challenging, as the comparative genomics field?and indeed, the larger genomics community?lacks consensus regarding meta-data collection and study documentation, particularly around sample and study design reporting. \textit{Opportunistic study collection.} Sample type, size, and collection techniques are critical study design considerations in comparative studies of primates. Until recently, flash frozen tissues were one of the only options for comparing primate biological material (REF Gallego Romero). Because of the difficulty of obtaining samples (both for logistic and ethical reasons), we could typically sample only a small number of individuals from each species. Consequently, some primate comparative studies only have only a handful of individuals per species (REF Pai et al Tomas? papers, Browand et al). Particularly problematic is when there is only 1 individual per species, as individual and tissue are confounded (REF tomas? papers). Even when there are 3 or more individuals per species, tissues are often subject to high environmental variances because the donors are in an uncontrolled environment, and also flash frozen and shipped post mortem. The necessity of collecting samples opportunistically, together with small sample sizes, can lead to incomplete power to detect regulatory differences between species in any given study, and hence to relatively large apparent differences between studies. Furthermore, it was nearly impossible to obtain multiple tissue samples from the same individual. For example, to date, there have been no published comparative studies in primates that have analyzed multiple tissues sampled from the same individuals across multiple species in a balanced design [30]. Consequently, regulatory differences between tissues are always confounded with regulatory differences between individuals. In turn, relative measures of tissue-specific regulatory differences between species are confounded with inter-tissue differences in regulatory variation within species. 	\section{Study design challenges in psychiatric genetics}
\textit{Confounders and other potential biases.} Unfortunately, these study design challenges are not limited to the field of primate comparative genomics. For example, these are issues in psychiatric genetics and are highly prevalent in the nascent field of eating disorders genomics. Confounders and other potential biases. When performing eating disorder studies, there are many differences between groups, both in comparing individual cases or across cases and controls. Problematically, many biological variables are confounded with the biological variable of interest, disease state. For example, in case-control studies of individuals with anorexia nervosa (AN), disease state is confounded with BMI. Even within cases, clinical variables can be confounded, including medication types, age of diagnosis, and number of rehospitalizations. In addition to the large number of biological variables, there are also technical considerations, such as RIN score. As discussed earlier, the larger genomics community lacks consensus regarding meta-data collection and study documentation. This area has also been discussed in the eating disorder field but have not been widely acted upon (REF Topher and Cindy?s methylation paper). \textit{Opportunistic study collection.}Similar to primates, tissues from humans are collected opportunistically. For many complex psychiatric trait, it is difficult to link genetic variation or gene regulatory differences, to phenotypes. While gene regulation in the brain is a logical place to start, it is difficult to access brains from living patients and a brain bank for eating disorders has only recently been established (REF).  [Another point you could mention here is that the brain is extremely heterogeneous ? which regions are most relevant? It?s difficult to know.] Moreover, while recent research suggests tissues besides the brain may be relevant to disease state in AN (Watson et al 2019), it is hard to decide which tissue to study. Blood is relatively easy to collect from individuals with eating disorders, but the clinical utility of whole blood is unknown. Since individuals are collected opportunistically, collecting large sample sizes, particularly in longitudinal studies are difficult. Furthermore, individuals are almost always recruited during a state of illness, so it can be difficult to predict patient trajectory and therefore the outcomes represented in the cohort. This issue is compounded by the fact that many eating-disorder phenotypes, including treatment outcomes, are nebulous (Kahlsa et al 2017). Indeed, although the field of eating disorder genomics faces many of the same obstacles as primate comparative genomics and psychiatric genomics more generally, it also presents a unique set of challenges. .

\section{Conclusion}Much of my thesis work has been devoted to addressing these challenges of potential biases and opportunistic study collection. in Chapters 2 and 3, I present examples of how to approach these challenges in gene regulatory studies across primates. Furthermore, I demonstrate that adherence to these key study design principles helps elucidate biological insight. In Chapter 4, I apply these lessons learned in a new context: comparing individuals with eating disorders at high risk of rehospitalization versus those with lower risk. In the final chapter, I discuss opportunities and challenges when using functional genomics to study eating disorders. 