\chapter{A comparative study of endoderm differentiation in humans and chimpanzees 
}\label{ch:tb-suscept}

\section[Abstract]{Abstract\footnotemark}

There is substantial interest in the evolutionary forces that shaped the regulatory framework in early human development. Progress in this area has been slow because it is difficult to obtain relevant biological samples. Inducible pluripotent stem cells (iPSCs) may provide the ability to establish in vitro models of early human and non-human primate developmental stages. Using matched iPSC panels from humans and chimpanzees, we comparatively characterize gene regulatory changes through a four-day timecourse differentiation of iPSCs  into primary streak, endoderm progenitors, and definitive endoderm. As might be expected, we find that differentiation stage is the major driver of variation in gene expression levels, followed by species. We identify thousands of differentially expressed genes between humans and chimpanzees in each differentiation stage. Yet, when we consider gene-specific dynamic regulatory trajectories throughout the timecourse, we find that at least 75\% of genes, including nearly all known endoderm developmental markers, have similar trajectories in the two species. Interestingly, we observe a marked reduction of both intra- and inter-species variation in gene expression levels in primitive streak samples compared to the iPSCs, with a recovery of regulatory variation in endoderm progenitors. The reduction of variation in gene expression levels at a specific developmental stage, paired with overall high degree of conservation of temporal gene regulation, is consistent with the dynamics of a conserved developmental process.

\footnotetext{Citation for chapter: Blake LE*, Thomas SM*, Blischak JD, Hsiao CJ, Chavarria C, Myrthil C, Gilad Y, Pavlovic
BJ. 2017. A comparative study of endoderm differentiation in humans and chimpanzees. Genome Biology 19(1):162. * denotes equal contribution.}

\section{Introduction}\label{ch03-introduction}

Differences in gene regulation between humans and other primates likely underlie the molecular basis for many human-specific traits \cite{RN1343}. For example, it has been hypothesized that human-specific gene expression patterns in the brain might underlie functional, developmental, and perhaps cognitive differences between humans and other apes \cite{RN1798, RN1799}. Providing a measure of support for this notion, a recent comparative study that explored the temporal dynamics of gene regulation found potential differences in the timing of gene expression in the developing brain across primates \cite{RN1909}. The authors argued that such differences might be related to inter-species differences in the timing of developmental processes. 
Other comparative studies of gene regulatory phenotypes in primates have resulted in important insights into the evolution of gene expression levels and the traits they are associated with \cite{RN1342}. Yet, we are also finding that comparative studies of gene expression patterns alone-- without additional context or perturbation-- provide relatively little insight into adaptive phenotypes. To a large degree, the challenge is that comparative studies in primates are extremely restricted because we only have access to a few types of cell lines and to a limited collection of frozen tissues \cite{RN1342}. Frozen post-mortem tissues are not optimal templates for many functional genomic assays; as a result, we lack datasets that survey multiple dimensions of gene regulatory mechanisms and phenotypes from the same individuals \cite{RN1339, RN1342}. Moreover, because it is rare to collect a large number of tissue samples from the same donor (particularly in non-human primates), we have never had the opportunity to study cross-species, population-level patterns of gene regulation in multiple tissues or cell types derived from the same genotype (same donor) in non human apes. 
Recent technological developments in the generation and differentiation of induced pluripotent stem cells (iPSCs) now provide a renewable, staged and experimentally pliable source of terminally differentiated cells. Utilizing timecourse differentiation protocols, we can examine the context dependent nature of gene regulation, as well as the temporal roles of gene expression as different cell types and developmental states are established \cite{RN1328}. This approach seems promising, and indeed, a handful of recent studies have been successful in utilizing iPSCs from humans and chimpanzees to characterize the uniquely human aspects of craniofacial development \cite{RN1374} and cortex development \cite{RN1395, RN1385}. 
Primate iPSC panels are a particularly attractive system for comparative studies of early development. Based on studies in model organisms, we expect differentiation into a germ layer in a mammalian system to be extremely conserved \cite{RN1384}. We thus set out to ask whether we can recapitulate a conserved early developmental gene regulatory trajectory in human and chimpanzee iPSC-derived cell lineages. Our rationale is that if we can demonstrate the fidelity of the iPSC model in this context, it would provide support for the notion that the iPSC system can be a useful tool for future comparative studies of dynamic biological processes.  
To this end, we chose to differentiate iPSCs from human and chimpanzee into the endoderm germ layer; from which essential structures in the respiratory and digestive tracts are ultimately derived. These structures include the liver, the pancreas, and the gall bladder, the lung, the thyroid, the bladder, the prostate, most of the pharynx and the lining of the auditory canals and the larynx \cite{RN1384}. Using this system, we found evidence to support developmental canalization of gene regulation in both species, 24 hours after differentiation from an iPSC state. 

\section{Results}

\subsection{Study design and data collection in the iPSC-based system}

To perform a comparative study of differentiated cells, we used a panel of 6 human and 4 chimpanzee iPSC lines previously derived and characterized by our lab \cite{RN1401, RN1396}. We differentiated the iPSCs into definitive endoderm, a process that was completed over 3 days \cite{RN1328}, and included replicates of cell lines that were independently differentiated (see Methods; Figure 1A). We performed the differentiations in two batches, with equal numbers of human and chimpanzee samples in each batch. We harvested RNA from iPSCs (day 0) prior to differentiation and subsequently every 24 hours to capture intermediate cell populations corresponding to primitive streak (day 1), endoderm progenitors (day 2), and definitive endoderm (day 3). Overall, we collected a total of 32 human samples and 32 chimpanzee samples (Figure 1A). We confirmed that RNA from all samples was of high quality (Additional file 1: Table S1; Figure S1) and subjected the RNA to sequencing to estimate gene expression levels. Detailed descriptions of all individual donors, iPSC lines, sample processing and quality, and sequencing yield, can be found in the Methods section and Additional file 1: Tables S2A-B. 
To estimate gene expression levels, we mapped reads to the corresponding genome (hg19 for humans and panTro3 for chimpanzees) and discarded reads that did not map uniquely \cite{RN1361}. We then mapped the reads to a list of previously described metaexons across 30,030 Ensembl genes with one-to-one orthology between human and chimpanzee \cite{RN1339, RN1397}. We eliminated genes that were lowly expressed in either species, removed data from one clear outlier sample (H1B at Day 0; Figure S2A), and normalized the read counts (see Methods; Figures S2B-C) to obtain TMM- and cyclic loess-normalized log2 counts per million (CPM) values for 10,304 orthologous genes (Figure 2A). These normalized gene expression values were used in all downstream analyses. 
Beyond the gene expression data we collected, in the second batch, we assessed the purity of the cell cultures at each day of the timecourse using flow cytometry with a panel of six canonical markers. These markers correspond to the cell types we expected in the different stages of differentiation (Figure 1B; Figure S3A-B). We assessed purity of the samples in the first batch of differentiation as well, but due to a technical problem with the antibodies, those values are not informative. Overall, the FACS-based estimated purity levels for the sample in the second batch are high and consistent across species in days 0 and 1 (\textless 0.79 and \textless 0.66, respectively; Figure 1B; Additional file 1: Table S3; Figures S3-S4). In days 2 and 3, however, purity levels were considerably lower in the human than in the chimpanzee samples (Figure 1B; Additional file 1: Table S3; Figures S3-S4). On the one hand, this inter-species difference in purity in the later stages of the time course limits the insight we can draw from this comparative study, as we discuss throughout the paper. On the other hand, because our main focus is to confirm that this early developmental process is conserved in the two species, the observation of strong conservation, which we discuss below, is robust with respect to this inter-species technical difference in purity.

\begin{figure}
\centering \includegraphics[width=5in]{img/ch03/Figure_1.pdf}
\caption[Study design and quality control analyses.]{\textbf{Study design and quality control analyses.} (A) Study design. Samples from four chimpanzees and six humans were studied at four time points during endoderm development. We included two technical replicates from each of the chimpanzees and two technical replicates for two of the six humans. iPSC induced pluripotent stem cell, PS primitive streak, EP endoderm progenitor, DE definitive endoderm. (B) Purity analysis. Cell type composition at each day based on FACS analysis (see Methods), estimated by k-means clustering. (C) Heat map of normalized log2(CPM) as a measure of expression levels of TFs that are known to be highly expressed in one or more stages in the differentiation to endoderm. Generally, samples from the same day, regardless of species, cluster together.}
\label{fig:limma}
\end{figure}

\subsection{iPSCs-based system effectively models primate endoderm differentiation}

Given the potential impact of study design properties on gene expression data and subsequent conclusions \cite{RN1402}, as a first step of our analysis, we confirmed that none of our recorded variables related to sample processing (apart from sample purity, as stated above) were confounded with our main variables of interest, namely day and species (see Methods; Additional file 1: Tables S4A-D; Figures S2C and S5). Once we were confident that our study design provided an effective data set for addressing our biological questions of interest, we performed a global survey of the gene expression data using principal component analysis. This analysis indicated that the primary sources of gene expression variation are differentiation day (Figure 2A; Additional file 1: Tables S4A-E; regression of PC1 by differentiation day, P \textless 10-15), followed by species (regression of PC2 by species, P \textless 10-15). This observation was also supported by clustering analysis based on the correlation matrix of pairwise comparisons of the gene expression levels (Figure S6).
After characterizing global gene expression patterns, we focused on the expression of specific transcription factors with known roles in developmental pathways (Figure 1C) and other previously known lineage specific markers \cite{RN1330, RN1328, RN1329}. Consistent with the results of our FACS analysis, we observed that the temporal trajectory of expression levels of known lineage specific markers and transcription factors further supported the assumed differentiation stages in each day (e.g. primitive steak-specific markers had increased expression on day 1, Figure 2B). The lineage specific markers and transcription factors were expressed at comparable levels in humans and chimpanzees at the relevant time points, consistent with previous literature \cite{RN1328, RN1329}, and further supporting the validity of our in vitro system (Figure S7A).

\begin{figure}
\centering \includegraphics[width=5in]{img/ch03/Figure_2.pdf}
\caption[General patterns in the data.]{ \textbf{General patterns in the data.} (A) Normalized log2(CPM) expression measurements for all genes projected onto the axes of the first two PCs. Color indicates day. Shape represents species. PC1 is highly correlated with differentiation day (r?=?0.92). PC2 is highly correlated with species (r?=?0.93). (B) Box plots of normalized expression values for genes with known roles in endoderm development.}
\label{fig:gwas}
\end{figure}

\subsection{Comparative assessment of gene expression changes during differentiation}

To identify gene expression differences between humans and chimpanzees throughout the timecourse, we used the framework of linear models (see Methods). We first assessed how many genes were differentially expressed (DE) between species at each time point independently. Using this approach, we classified thousands of genes as DE between the species (at FDR of 5\% 4475 ? 5077 genes are classified as inter-species DE at different times points; Figure 3A; Additional file 1: Tables S5A-D). Even at a fixed FDR cutoff, nearly half of the genes that were classified as DE between the species at any single time point were found to be DE in all time points (2269 genes). Nearly a third of genes whose expression was measured in our experiment were not classified as DE between the species at any time point (2862 genes, 28\%). 
We proceeded to consider temporal expression patterns within species. When analyzing expression changes across consecutive time points, we had more power to detect temporal gene expression differences in chimpanzee compared to humans (Figures 3B-C and 4; Additional file 1: Tables S6A-F), especially with respect to the transition between endoderm progenitors (day 2), and definitive endoderm samples (day 3). This property is likely related to the inter-species difference in purity of samples from these days, as discussed earlier. When we accounted for incomplete power (see ?Methods?), we found a remarkably consistent pattern whereby 77\% of DE genes between iPSCs and primitive streak in humans are also DE between these states in chimpanzees; similarly, 77\% of DE genes between primitive streak and endoderm progenitors in humans are also DE between these states in chimpanzees; and 80\% of DE genes between endoderm progenitors and definitive endoderm in humans are also DE between these states in chimpanzees (Additional file 1: Table S7). As might be expected from these observations, we found that the relationship between day and gene expression was largely independent of species (Figure 3D; Additional file 1: Table S8A-C).

\begin{figure}
\centering \includegraphics[width=5in]{img/ch03/Figure_3.pdf}
\caption[Number of DE genes in pairwise analyses.]{ \textbf{Number of DE genes in pairwise analyses. Venn diagrams of.} (A) DE genes at each day, (B) DE genes between consecutive time points in humans, (C) DE genes between consecutive time points in chimpanzees, (D) genes with a significant species?time point interaction effect at each day (DE was classified at FDR of 5\% in all cases).}
\label{fig:classifier}
\end{figure}

\begin{figure}
\centering \includegraphics[width=5in]{img/ch03/Figure_4.pdf}
\caption[High sharing of DE genes across species.]{\textbf{High sharing of DE genes across species.}  A circos diagram with the number of shared, human-specific, and chimpanzee-specific DE genes across time points. There is a high degree of sharing of DE genes (yellow ribbon), particularly from day 0 to 1 and day 1 to 2.}
\label{fig:classifier2}
\end{figure}

\subsection{Joint Bayesian analysis reveals conservation of temporal gene expression profiles}

In an attempt to further overcome issues of incomplete power affecting these original naïve pairwise DE comparisons, and to account for dependency in data from different time points, we utilized a Bayesian clustering approach implemented by Cormotif \cite{RN1908}. This joint modeling technique leverages expression information shared across time points to identify the most common temporal expression patterns (referred to as ?correlation motifs?). 
We identified diverse expression patterns that emerge as differentiation progresses in both species (Figure 5; Additional file 1: Table S9A) as well as a set of 3789 genes whose expression is not significantly altered throughout the timecourse (8004 genes could be reliably classified into a motif, Additional file 1: Table S9A; Figure S8A). This analysis revealed further evidence for conserved gene expression patterns, as 75\% of genes assigned were assigned to motifs with the same or similar temporal regulatory trajectories in both species (Figure 5). Further, when we excluded data from the definitive endoderm samples, where we suspect that a particularly large inter-species difference in sample purity has increased gene expression variance between the species, we assigned 85\% of genes to motifs with the same temporal trajectories across species. These observations are robust with respect to the number of correlation motifs, the method used to combine data from technical replicates, which days were included in the pairwise comparisons, and the inclusion of all 10,304 genes in the analysis (Figures S8B-D and S9). Our observations are also robust with respect to the overall approach used to estimate and compare gene expression trajectories (Figure S7B).
We found two correlation motifs with a potential marked difference between the species at a given stage (motif 4 with 187 genes and motif 7 with 686 genes). In both of these motifs, data from the earliest time points were conserved but gene regulation in the final stage (day 2 to 3) differed between the species. The genes in these motifs were enriched for Gene Ontology (GO) annotations related to animal organ development (e.g. \textit{NRTN, PITX2, RDH10}), anatomical structure morphogenesis (\textit{ARHGDIA, EHD2, SERPINE1}), regulation of developmental process (\textit{FLRT3, LOXL2, SEMA7A}), and regulation of cell differentiation (\textit{DIXDC1, ENC1, IRF1}; Additional file 1: Table S9B) \cite{RN1783}. These four GO annotations were not enriched in other similarly sized motifs or group of motifs (Additional file 1: Tables S9C-E) \cite{RN1783, RN1794}. Unfortunately, due to interspecies differences in purity at the later days, we cannot definitively determine whether these enrichments are driven by biological or technical differences. 

\begin{figure}
\centering \includegraphics[width=5in]{img/ch03/Figure_5.pdf}
\caption[Gene expression motifs.]{\textbf{Gene expression motifs.} Correlation motifs based on the probability of differential expression across days for each species with the number of genes assigned to each correlation motif. The shading of each box represents the posterior probability that a gene is DE between two time points in a given species. Each row (``correlation motif?) represents the most prevalent expression patterns. Out of 10,304 genes, 8004 were assigned to one correlation motif in this model.}
\label{fig:classifier3}
\end{figure}

\subsection{Reduced variation in gene expression levels at primitive streak}

We next turned our attention to differences in the magnitude of variation in gene expression levels across time points, within and between species. Previous studies reported that variation in gene expression levels between individuals was lower in iPSCs than in differentiated cells (\cite{RN1399, RN2196} and Figure S10A). We were thus interested in gene expression variation during iPSC differentiation in our comparative system. 
We first compared within-species expression variation for all 10,304 orthologous genes across time points. We found a reduction in inter-individual variation of gene expression levels as the human samples differentiated from iPSCs to primitive streak (P \textless 10-15, Figure 6). We also detected this pattern when we considered the chimpanzee samples (P \textless 10-15, Figure 6), but the effect size in chimpanzee is much smaller. We did not identify similar reduction in variation in gene expression levels in any other transition during the timecourse in either species (P \textgreater 0.5 for testing the null of no change in variance of gene expression from day 1 to 2 and from day 2 to 3 in each species). As mentioned above, the purity of the samples in days 2 and 3 is lower than that of samples in days 0 and 1, though we only successfully measured the purity values for the samples that were processed in the second batch. We accounted for the measured purity values by regressing them out of the gene expression data. As might be expected, accounting for the purity values resulted in different patterns observed for the data from days 2-3. Yet, the reduction in variation from day 0 to 1 persisted in both species even after we accounted to the purity values (Figure S10B).
We turned our attention to regulatory divergence between species. The overall human-chimpanzee divergence in gene expression levels was also slightly reduced as samples differentiated from iPSCs to primitive streak (Mann-Whitney U Test, P = 0.04; Figure S10C), but not in any other transition during the timecourse. Furthermore, while we classified 504 genes as DE between humans and chimpanzees exclusively in iPSCs (of a total of 4,475 DE genes in iPSCs, FDR = 5\%; Figure 3A; Additional file 1: Table S10), we found only 279 genes that were DE exclusively in primitive streak samples (from a total of 4,408 DE genes for the primitive streak; Figure 3A; Additional file 1: Table S10). The number of genes that are DE between the species exclusively in endoderm progenitors and definitive endoderm samples is higher (at FDR of 5\%, 402 and 934, respectively; Figure 3A; Additional file 1: Table S10). The observation of a smaller number of genes that are DE exclusively in primitive streak samples compared with iPSCs is robust with respect to normalization method, purity of the samples, FDR cutoff, and differentiation batch (Additional file 1: Table S10; Figures S2C, 10C and S11). While the difference in divergence and the number of DE genes between these differentiated states is modest, and could potentially be explained by a number of non-biological factors (including the differences in purity in the later day), this observation was intriguing to us.

\begin{figure}
\centering \includegraphics[width=5in]{img/ch03/Figure_6.pdf}
\caption[Global reduction of variation in gene expression from the iPSCs to primitive streak state.]{\textbf{Global reduction of variation in gene expression from the iPSCs to primitive streak state.} Box plot of the log2 variance of expression levels for each gene. Variation in gene expression levels are significantly reduced from iPSCs to primitive streak (P \textless 10-15 in both species) but not in subsequent time points (P \textgreater 0.5 in both species).}
\label{fig:classifier3}
\end{figure}

We thus focused on the transition between iPSCs to primitive streak in both species. The recorded technical factors (including purity for these states) are highly similar across biological conditions in days 0 and 1, and therefore are not likely to explain this observation (Additional file 1: Table S11). We thus proceeded to analyze the trajectory of variation in expression level on an individual gene basis. In this analysis, we were particularly interested to address whether the individual genes that undergo a change in variation of expression levels are shared across species. An observation of excess sharing could not be explained by inter-species differences in technical factors and hence would provide substantial support to the notion of conserved reduction of regulatory variation as the samples begin their differentiation process.
We used F tests to identify genes whose within-species variation in expression levels differs across time points (see ?Methods?). Distributions of P values from all tests can be found in Figures 7A-B and Additional file 1: Tables S12A-B, which indicate that for a large number of genes, within-species variation in expression levels were reduced exclusively in primitive streak samples. Indeed, while we did not have much power to detect differences in variation of individual gene expression levels between states (due to the small number of individuals in each species), we observed a clear excess of small P values when we tested the null hypothesis that there was no reduction in gene expression levels from day 0 to 1. Using Storey?s approach \cite{RN1393} to account for incomplete power, we estimated that within-species variation in expression levels was reduced as the samples differentiate from iPSCs to primitive streak in 83\% and 27\% of human and chimpanzee genes, respectively (Figures 7A-B). This result was robust with respect to the method used to calculate the proportion of true positives \cite{RN1392} (Figure S12). We did not observe reduced variation of gene expression in any other differentiation state in our data (Figures 7; S13A).
We next asked about the overlap of genes with reduced variation in primitive streak samples across the two species. Specifically, we asked whether human genes with lower within-species variation in expression levels in primitive streak are more likely to show the same pattern in chimpanzee genes. For this analysis, we again used the Storey approach \cite{RN1393} to estimate the proportion of true positive tests in one species, conditional on the observation of reduced variation in the other species (see Methods). We estimated that 47\% of genes whose variation in expression level is reduced in human primitive streak samples showed a similar pattern in chimpanzees (under a permuted null we expect 27\%, P \textless 10-4, Figure 7C; Additional file 1: Table S13). When we condition on observing a reduction of variation in chimpanzees, the overlap with humans was 84\% (under a permuted null we expect 83\%, P = 0.38; Figure 7D). This high value was not unexpected because of the initial large proportion of human genes with a clear signature of reduced variation in primitive streak. Because any technical differences that are confounded with species would contribute to increased inter-species differences, these observations support, yet probably underestimate, the degree of high conservation of regulatory patterns in humans and chimpanzees. 
Using a similar approach to account for incomplete power, we also found a marked overlap of genes whose expression underwent a significant increase in variation throughout the transition from primitive streak to endoderm progenitors (Figures 7E-F; Figure S13B). All our observations were robust to a wide range of statistical cutoffs used to classify genes whose within-species variation changes across the differentiation states (Figures S14-S16). In particular, because these are reports of conserved patterns, they are likely to underestimate the degree of conservation given technical differences between the species, including the difference in purity of the samples in days 2 and 3.
Finally, we sought to provide insight into the potential functional consequences of our observations. Interestingly, genes that show reduction of variation in gene expression levels in both species are enriched for GO annotations \cite{RN1783} related to development, including neuroepithelial cell differentiation (e.g. \textit{MYCL, NODAL, CDH2}), cell migration during gastrulation (\textit{FGF8, MIXL1}), and trophectodermal cell differentiation (\textit{CNOT3, EOMES, SP3}; Additional file 1: Table S14). Moreover, we found that null mutation in mouse orthologs \cite{RN2197} of the primate genes with shared reduction of expression variation in our study were more highly associated with embryonic lethality than null mutation in mouse orthologs of primate genes that did not show that pattern (41\% versus 28\%, P \textless 10-4 \cite{RN2198, RN1797}). 


\begin{figure}
\centering \includegraphics[width=5in]{img/ch03/Figure_7.pdf}
\caption[Conserved patterns of reduced variation in gene expression at primitive streak.]{\textbf{Conserved patterns of reduced variation in gene expression at primitive streak.} We plotted the P value distributions of F tests of the null hypothesis that there is no reduction in variation in gene expression levels as samples progress along the time course in human (A) and chimpanzee (B) samples. Pi zero is the estimated proportion of null tests in each distribution. In the next four panels, we plotted the P value distribution for the same test, but included only genes whose variation was classified as reduced between states in the other species; thus in (C) we plotted P value distributions of F tests in chimpanzees only for genes who variation was classified as reduced (P \textless 0.05) in humans and in (D) we did the reverse. In (E) (chimpanzee conditional on human) and (F) (human conditional on chimpanzee), we plotted the P value distributions of F tests of the null hypothesis that there is increase in variation in gene expression levels as the samples progress from day 1 to 2.}
\label{fig:classifier4}
\end{figure}

\section{Discussion}

Our results indicate a strongly conserved temporal expression profile across species during early differentiation. We observed a large number of genes with similar expression profiles across species. Indeed, we found that DE genes between differentiation states are shared between the two species far beyond what expected by chance alone. When we jointly analyzed data from the entire timecourse, nearly all the gene expression trajectory motifs we identified, including 75\% of all genes assigned to a motif, are shared across the two species (Figure 5). Still, our observations likely underestimated the proportion of shared regulatory patterns due to incomplete power.
Our finding that regulatory trajectories throughout endoderm differentiation are generally highly conserved in these two species was expected. Yet, our observation that a large number of genes are associated with conserved reduced regulatory variation in a specific transition state is a somewhat surprising property. Indeed, in our opinion, the most significant finding of this study is the observation that regulatory variation is reduced in both humans and chimpanzees as the cell cultures differentiate from iPSCs to primitive streak. We found a marked overlap between the species in the specific genes that experience this reduction of regulatory variance, indicating high degree of conservation in this process.
Before we discuss the potential implications of our observations, we will first discuss a few considerations regarding the iPSC based differentiation models. We argue that the use of iPSC models allows for greater control and transparency of comparative studies in primates, including a better appreciation of caveats that have always affected such studies but were typically cryptic. 
For more than a decade, comparative genomics in primates has relied on the use of frozen tissues. An implicit assumption underlying the use of these tissues has been that they faithfully reflect interspecies gene regulatory similarities and differences. Yet, we know that gene regulation in these tissues was likely impacted by non-genetic factors, such as the individual?s diet, age, and cause of death. In addition, it is nearly impossible to stage frozen tissues with respect to cellular composition. In fact, in practically all studies of comparative data from frozen tissues, cellular composition was not even measured. Indeed, comparative studies of gene expression in primate frozen tissue samples, including those by our own group, simply assumed that most observed patterns are driven by genetic control. 
In contrast to frozen tissues, the primate iPSC-based differentiation model allows us to minimize the impact of non-genetic (e.g. environmental) factors. We can also control to a large extent the cellular composition of the samples, and more importantly, we can measure and account for differences in cellular composition. Comparative experiments with iPSC-based differentiation are certainly not flawless, but they allow us to more explicitly characterize, and often account for, confounding factors than it was possible with frozen tissue samples. In the case of the current study, we used human and chimpanzee iPSC lines that were generated, and differentiated, using the same protocols. We made considerable efforts to balance the majority of sample processing properties related to our study design with respect to species and time point. For example, the two differentiation batches we used included multiple human and chimpanzee lines (which we also balanced with respect to gender). Admittedly, in vitro differentiation protocols are not identical to natural developmental signaling and natural cell-driven developmental processes may be overridden by our administered media conditions. Moreover, although we used an identical differentiation protocol across the entire experiment, we observed a wide distribution of cell purity across samples, with a marked difference between the species in the purity of cultures form days 2 and 3. 
 The fact that we are able to measure and discuss purity and cellular composition as a potential flaw in our study is an advantageous property of the iPSC-based differentiation system. At present, we cannot exclude the possibility that the observed differences in cellular heterogeneity across time points may have driven the observation of reduced regulatory variation exclusively in primitive streak samples. In our opinion, however, this is unlikely, though our arguments are mostly circumstantial and we will not be able to provide a definitive answer without additional single cell experiments (which will be the scope of a future study). Our intuition is based on the following properties: First, the iPSC cultures are the most homogenous in our experiment and we nevertheless observe a reduction of variation in gene expression levels in day 1 samples. Second, the conclusion of high sharing and similarities between species should be robust (conservative, in fact) with respect to technical differences between species, including in purity. Third, when we account for the purity values measured for the second batch of samples, the relative expression patterns remain similar.
That said, to provide definitive answer, we will revisit these questions with an experimental design that involves single cell data from the same comparative differentiation trajectory. Single cell RNA sequencing data will also be able to shed light on our observation that gene regulation in definitive endoderm (day 3), unlike earlier days, does not indicate strong conservation between the species. In definitive endoderm samples, we observed the largest number of DE genes across species, the smallest number of DE genes between two consecutive time points in humans, and the lowest overlap in DE genes between time points. Unlike in the human samples, the chimpanzees had a relatively consistent number of DE genes between any two consecutive time points. Some of these observations might be explained by the cell purity difference between species. 
It should be noted that the effect sizes for the reduction in variation from day 0 to 1 are small in the chimpanzee samples and that the P value distributions do appear quite different across species. These differences, however, do not invalidate our finding that there is a significant overlap of genes that undergo a reduction of variation in gene expression levels in both species. If the reduction of variation in gene expression levels were spurious within each species, then we would not expect a statistically significant overlap of such genes. The observation of reduced regulatory variation is rather unusual in general, partly due to the unusual design of our study. Indeed, only few comparative studies have been designed to allow one to measure changes in variation over time. One such example, from a completely different context, can be found in a previous study in which monocytes from humans, chimpanzees, and rhesus macaques, which were stimulated with lipopolysaccharide (LPS) to mimic infection \cite{RN1400}. When comparing gene expression in LPS-stimulated monocytes to that of non-stimulated cells, the authors found a reduction of inter-species variation in gene expression levels in a number of key transcription factors involved in the regulation of TLR4-dependent pathways. 
In our study, we found enrichment for genes in developmental pathways among genes with conserved reduction of expression variation. This observation is consistent with the notion \cite{RN1386} that developmental pathways need to be tightly regulated in general. This notion is also supported by deep conservation and lethality upon disruption seen in many of these genes. Null mutations in mouse orthologs of over 40\% of the genes with conserved reduced regulatory variation at primitive streak are associated with embryonic lethality. For example, \textit{Xenopus laevis} embryos with null mutations in the ortholog of human \textit{MIXL1} exhibit abnormalities in primitive streak and node formation \cite{RN1553}. Similarly, homozygous null \textit{MIXL1} mice have abnormal mesoendoderm development and do not survive to birth \cite{RN1646}. \textit{EOMES} homozygous null mice lack trophoectoderm outgrowth \cite{RN1760} and do not properly form the definitive endoderm \cite{RN1772}. This mutation is lethal early in gestation. Overall, regulation of these genes is likely to be finely tuned at early development. 
 Indeed, reduced regulatory variation early in the endoderm differentiation process may be driven by the property of canalization during development. The theory of canalization posits that developmental processes end in a finite number of states despite minor environmental perturbations \cite{RN1391, 1386, 1387}. Canalization is fundamentally linked to evolutionary states \cite{RN1387}, and thus phenotypic robustness; therefore, even when reduced variation in gene expression levels is observed in cell culture, the explanation of canalization is intuitively appealing considering the discrete nature of cell types in an adult animal. Our results suggest that stages subsequent to primitive streak may follow a more relaxed transcriptional regulation with higher influence of individual genotypes. 
Our observations may be consistent with activation of deeply conserved regulatory programs at the initial stages of gastrulation followed by processes less affected by evolutionary constraint and therefore potentially more amenable to adaptation. In other words, our results supports the expectation that gastrulation is a highly canalized and conserved process in humans and chimpanzees. More generally, we believe that despite limitations to studying comparative development using iPSC models, which we have discussed, this system provides the opportunity to study previously unappreciated aspects of primate biology. 

\section{Methods}

\subsection{Human and chimpanzee iPSC panels}

In this study, we include four chimpanzee iPSC lines (2 males, 2 females) from a previously described panel \cite{RN1396} and six human lines (3 males, 3 females) \cite{RN1401} matched for cell type of origin, reprogramming method, culture conditions and closely matched to passage number (median passage was within 1 passage across species and differentiation batches). We evaluated iPSC lines for pluripotency measures, differentiation potential, lack of integrations and normal karyotypes as described previously \cite{RN1401, 1396} (Figures S17-S18). We identified one human individual (H5) that tested positive for episomal vector sequence (Figure S18). This individual was not an obvious outlier in any of our data (Figures 1B-C and 2), thus we choose to include it in our study. Original chimpanzee fibroblast samples for generation of iPSC lines were obtained from the Yerkes Primate Center under protocol 006-12. Human fibroblasts samples for generation of iPSC lines were collected under University of Chicago IRB protocol 11?0524. Feeder free iPSC cultures were initially maintained on Growth Factor Reduced Matrigel using Essential 8 Medium (E8) as previously described. After 10 passages in E8, all cell lines were transitioned to iDEAL feeder free medium that was prepared in house as specified previously \cite{RN1334}. Cell culture was conducted at 37°C, 5\% CO2, and atmospheric O2.

\subsection{Endoderm Differentiation}

To produce definitive endoderm and intermediate cell types, we followed a recently published three-day protocol that systematically identified and targeted pathways involved in cell fate decisions, at critical junctures in endoderm development \cite{RN1328} with minimal modification. At 12 hours prior to initiating differentiation, iPSC lines at 70-90\% confluence were seeded at a density of 50,000 cells/cm2. Basal medial for differentiations consisted of 50/50 IMDM/F12 basal media supplemented with 0.5 mg/mL human albumin, 0.7 g/mL Insulin, 15 g/mL holo-Transferrin, 1\% v/v chemically defined lipid concentrate, and 450 uM 1-thioglycerol (MTG). For differentiation, basal media was supplemented with the following: day 0 to day 1 (Primitive streak induction) media included 100 ng/mL Activin A, 50 nM PI-103 (PI3K inhibitor), 2 nM CHIR99021 (Wnt agonist), days 1->2 (total of 2 media changes) media included 100 ng/mL Activin A and 250 nM LDN-193189 (BMP inhibitor). Two independent differentiation batches were performed, resulting in replicates for a subset of individuals (Additional file 1: Table S2). Each chimpanzee was replicated, while only two humans individuals were replicated across the two batches. Replicates were sex-balanced both within and across species. Cell culture was conducted at 37°C, 5\% CO2, and atmospheric O2.

\subsection{Purity assessment using flow cytometry}

Cells were dissociated using an EDTA based cell release solution, centrifuged at 200 x g for 5 minutes at 4C and washed with PBS. Subsequently, 0.5-1 million cells were fixed and permeabilized using the Foxp3 / Transcription Factor Staining Buffer Set from eBioscience. Cells were fixed at 4C for 30 minutes before washing once using FACS buffer (autoMACS® Running Buffer, Miltenyi Biotech). 150,000 cells were transferred to BRAND lipoGrade 96 well immunostaining plates and centrifuged at 200 x g for 5 minutes at 4C. Cells were rinsed in FACS buffer then resuspended in the staining solution. A single master mix containing 1X Permeabilization buffer (eBioscience), BD Horizon Brilliant Stain Buffer and antibodies was prepared and 30 ?L of this mix was added to each well containing cells. 
In order to estimate purity for each day of the timecourse, we utilized a mixture of six different directly labeled antibodies: OCT3/4 (BV421 labeled clone 3A2A20, Biolegend), SOX2 (PerCP-Cy5.5 labeled clone O30-678, BDbio), SOX17 (Alexa 488 labeled clone P7-969, BDbio), EOMES (PE-Cy7 labeled clone WD1928, eBioscience), CKIT (APC labeled clone 104D2, Biolegend), CXCR4 (BV605 labeled clone 12G5, Biolegend). All antibodies were used at the manufacturer recommended dilution except CKIT and CXCR4, which was used at 1/10 of the manufacturer specified concentration (15 ng of each antibody in final volume of 30 ?L per staining). We found that the manufacturer recommended dilution produced acceptable results for live cells, however, upon fixing, we observed nonspecific binding by all populations. Thus we determined the optimal antibody titer to maximize the separation between iPSCs (biological negative) and Day 3 definitive endoderm. We found this optimal concentration to be in concordance with that quantity specified by a previous publication using the same antibody clone from a different manufacturer \cite{RN1338}. Cells were stained for 1 hour at 4C and subsequently washed 3x using a solution of BD Horizon Brilliant Stain Buffer containing 1X Permeabilization buffer, on the final wash cells were resuspended in 100 ?L FACS buffer for acquisition on a BD LSR II flow cytometer. 
After data acquisition compensation, we used the program FlowJo (http://docs.flowjo.com/d2/credits-2/) to determine scaling. To do so, we used data from single stained compensation beads (Life Technologies) that were stained and collected in parallel. Live, intact, single cells were gated based on FSC and SSC channels as previously described \cite{RN1328}. Day 0 iPSC purity was estimated by dual positive OCT3/4 and SOX2 \cite{RN1335} as well as negative staining for EOMES. Day 1 primitive streak purity was estimated primarily based on EOMES Positive staining \cite{RN1336, RN1329} but also negative staining for SOX17. Day 2 endoderm progenitor purity was quantified by positive staining for SOX17 expression \cite{RN1337} (CKIT could also be used, as its level peaks at day 2) and negative staining for CXCR4. Finally, day 3 definitive endoderm purity was estimated by double staining for CKIT and CXCR4 \cite{RN1338}. For all time points, cells were stained with the full complement of markers; initial gates were defined using fluorescence intensity levels of an iPSC line as a biological negative control for days 1, 2, and 3. For day 0 (iPSCs), a definitive endoderm time point was used to quantify the biological negative for OCT3/4 and SOX2 fluorescence intensity. All iPSC lines regardless of species were at comparable fluorescence intensity levels, so we choose a representative chimp and human line to use as our standard for defining and refining all gates. 
Fully resolving all time points simultaneously required us to define high and low staining gates, which were determined using the time points for that marker?s maximum and minimum fluorescence intensities. All gates were refined using the same two representative chimpanzee and human lines as used for determining biological negatives, resulting in one universal gating scheme that was applied to both species and all time points. A complete gating scheme is outlined in Figure S3A, with the final purity results for the second batch of differentiation in Additional file 1: Table S3. The samples in the first differentiation batch demonstrated hallmarks of improper fixing (highly nonspecific staining of antibodies, most notably for surface markers CXCR4 and CKIT), thus we were unable to determine reliable purity estimates for the first differentiation batch.
Purity was also determined using k-means clustering of minimally preprocessed flow cytometry data. After applying the same live, intact, single cell gating scheme used above, we exported compensated fluorescence channel values for processing in R. First, we visually inspected population separation by performing Principal Components Analysis (PCA) on compensated fluorescence channel values. We randomly sampled 1000 cells from each individual at each day, resulting in a total of 4000 randomly sampled cells per individual and 31,000 cells overall (no data was collected for individual H4 at day 3). To assign cells to a specific developmental day, we performed K-means clustering using K = 4 on a correlation matrix representing pairwise correlations between all 31,000 single cells. Four relatively well separated populations were visible after projection onto the first three principal components (Figure S4, top panel) and cluster assignment on the reduced data is shown in Figure S4, middle and bottom panels. We estimated cellular composition at each day by using the PCA to match clusters to a day assignment (Figure 1B).

\subsection{RNA extraction, library preparation, and sequencing }

We collected RNA from iPSCs (day 0) prior to adding day 1 media, and then every 24 hours during the differentiation timecourse for a total of 4 time points representing intermediate cell populations from iPSCs to definitive endoderm (Figure S6B). We extracted the RNA using the ZR-Duet DNA/RNA MiniPrep kit (Zymo) with the addition of an on column DNAse I treatment step prior to RNA elution. We used non-strand specific, polyA capture to generate RNA-seq libraries according to the Illumina TruSeq protocol. To estimate the RNA concentration and quality, we used the Agilent 2100 Bioanalyzer (Additional file 1: Table S1; Figure S1). We added barcoded adaptors (Illumina TruSeq RNA Sample Preparation Kit v2) and sequenced the 50 base pair single-end RNA-seq libraries on the Illumina HiSeq 4000 at the Functional Genomics Core at University of Chicago on two flowcells (Additional file 1: Table S2). To minimize the introduction of biases due to batch processing, we chose the RNA extraction batches, library preparation batches, sequencing pools, adaptor names, and flowcells in a manner that maximally partitioned the biological variables of interest (day, species, cell line; Additional file 1: Tables S2A and S4). 
We generated a minimum of 14,424,520 raw reads per sample. We used FastQC (http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) to confirm that the reads were high quality. 

\subsection{Quantifying the number of RNA-seq reads from orthologous genes}

We mapped human reads to the hg19 genome and chimpanzee reads to panTro3 using TopHat2 (version 2.0.11) \cite{RN1361}, allowing for up to two mismatches in each read. We kept on only reads that mapped uniquely. To prevent biases in expression level estimates due to differences in mRNA transcript size and the relatively poor annotation of the chimpanzee genome, we only kept reads that mapped to a list of orthologous metaexons across 30,030 Ensembl genes available for the hg19 and panTro3 genomes as described previously \cite{RN1339}.  Gene expression levels were quantified using the feature counts function in SubRead 1.4.4 \cite{RN1340}. For one sample (C2B at Day 0), the number of raw reads was approximately double the second highest number of raw reads. Therefore, we subsampled the raw reads to approximately the same number of raw reads as the second highest sample. 
We performed all downstream processing and analysis steps in R (version 3.2.2) unless otherwise stated. 

\subsection{Transformation and normalization of RNA-sequencing reads}

After receiving the raw gene counts, we calculated the log2-transformed counts per million (CPM) for each sample using edgeR \cite{RN1394}. To filter for the lowly expressed genes, we kept only genes with an expression level of log2(CPM) \textgreater 1.5 in at least 16 samples per species \cite{RN1367}. For the remaining genes, we normalized the original read counts using the weighted trimmed mean of M-values algorithm (TMM) \cite{RN1367} to account for differences in the read counts at the extremes of the distribution and calculated the TMM-normalized log2-transformed CPM. 
When we performed principal components analysis (PCA) using the TMM-normalized log2-transformed log2(CPM) values, we found one outlier (H1B at Day 0, Figure S2A). We removed this sample from the list of original gene counts.  We filtered for the lowly expressed genes by retaining genes with an expression level of log2(CPM) \textgreater 1.5 in at least 15 human samples and at least 16 chimpanzee samples. 10,304 genes remained. We performed TMM-normalization and then performed a cyclic loess normalization with the function normalizeCyclicLoess from the R/Bioconductor package limma \cite{RN1377, RN1398}. We found that the TMM-normalized log2(CPM) values were highly correlated with the TMM- and cyclic loess-normalized log2(CPM) values (r \textgreater 0.99 in the 63 samples; Figure S2B). We used the TMM- and cyclic loess-normalized log2(CPM) expression values in all downstream analysis unless otherwise stated. 
We calculated normalized log2-transformed RPKM values by using the function rpkm with normalized library sizes from the package edgeR \cite{RN1394} (Figure S2C). We measured the ``gene lengths? as the sum of the lengths of the orthologous exons and were also used in \cite{RN1396}. This method of calculating RPKM was highly correlated with a method in which we subtracted log2(gene length in kbp) from the TMM- and cyclic loess-normalized log2(CPM) values (r \textgreater 0.97). 

\subsection{Data quality and analysis of technical factors}

To assess the data quality, we performed Principal Components Analysis (PCA) on the normalized log2(CPM) values from above (Figure 2A). Principal component (PC) 1 was highly associated with day and PC2 was highly associated with species (r  \textgreater 0.92 for each, Figure 2A; Additional file 1: Tables S4A-B). We sought to determine if the study?s biological variables of interest were confounded with any of the study?s recorded technical aspects (Additional file 1: Table S4C-D).  First, we calculated which of our 35 recorded technical factors were statistically significant predictors of PCs 1-5 with individual linear models for each technical factor. The 19 statistically significant predictors (FDR cutoff of 10\% assessed on the 5x35 matrix) were carried to the second stage. In this stage, we determined which technical factors were associated specifically with either day or species, with individual linear models for each technical factor. We quantified these associations using the P values from analysis of variance (ANOVA) for the numerical technical factors and from Chi-squared test (using Monte Carlo simulated P values) for the categorical technical factors. Statistical significance was determined by Benjamini-Hochberg adjusted P value \textless 10\% (assessed on the 2x19 matrix). A variable for cell line include a species but not a day component and was tested in this pipeline. They were found to each be confounded with species (B.H. adj. P value  \textless 10-4) but not day (B.H. adj. P value  \textgreater 0.9), thereby increasing the confidence in our pipeline.  
We note that when all sequencing pools (mastermixes) were considered together, there was a relationship between adaptor sequence and day (Chi-squared test, Benjamini-Hochberg adjusted P = 0.01); however, this relationship is substantially weaker when ``adaptor sequence'' and ``day'' were tested in each of the 4 sequencing pools separately (B.H. adj. P  \textgreater 0.9 in each test).  Our most highly dependent variables with day or species were related to properties inherent to the iPSC model, including harvest density and day (B.H. adj. P = 0.02), harvest density and species (B.H. adj. P = 0.03), harvest time and day (B.H. adj. P = 0.01) (Additional file 1: Table S4D; Figure S5). 
During this analysis, we observed that the purity estimates were relatively similar across days and between species until the final day (Figure 1B; Additional file 1: Table S5; Figure S3C). Therefore, it was important to explore how the variance for a given technical factor was partitioned across the biological variables of interest (e.g. across the days, species, and day-by-species interactions). For each recorded technical variable, we created a reduced model and a full model. The reduced model contained only species and day as fixed effects and the technical factor as the response variable. The full model had the same response variable but contained species, day, and a species-by-day interaction as fixed effects. We then compared the two models and reported the significance (Additional file 1: Table S12). 
The exact tools used to compare the two models were data-dependent (Additional file 1: Table S2 and columns 1-2 in Table S12). For numerical data (24 technical factors), we constructed the full and reduced normal general linear models for each technical factor. We compared the models using ANOVA, and extracted the P value directly from ANOVA. For categorical data with 2 levels (3 technical factors), we constructed the two general linear models from the binomial family. We used ANOVA to compare the models and extracted the deviance along with its degrees of freedom. Based on the deviance, we calculated the Chi-Squared statistic and associated P value. 8 technical factors (such as RNA extraction data) contained categorical data with more than two levels. We modeled this data type with multinomial logistic regression with the R/Bioconductor package nnet \cite{RN1383} and used ANOVA to obtain the likelihood ratio statistic and associated P value.  We performed this process for each technical factor using data from days 0 and 1 as well as from days 0 to 3 (Additional file 1: Table S12).


\subsection{A linear model based framework to perform pairwise differential expression analysis }

Differential expression was estimated using a linear model based empirical Bayes method implemented in the R package limma \cite{RN1368, RN1369}. In order to use a linear modeling approach with RNA-seq read counts, we calculated weights that account for the mean-variance relationship of the count data using the function voom from the limma package \cite{RN1341}. This limma and voom pipeline has previously been shown to perform well with n > 3 biological replicates/condition \cite{RN1370, RN1371}. 
For all pairwise differential expression comparisons, the species, day, and a species-by-day interaction were modeled as fixed effects, and individual as a random effect. Individual (cell line) rather than differentiation batch was modeled as a random effect because when using a linear model, individual was most highly correlated with PCs 2 and 3, whereas batch was most highly correlated with PC 10. Since our recorded technical factors were not confounded with our biological variables of interest and did not contribute significantly to the first five principle components of variation (Additional file 1: Tables S4A-E), we did not include any other covariates. 
We used contrast tests in limma to find genes that were differentially expressed (DE) by species at each day (Additional file 1: Table S5), DE between days for each species (Additional file 1: Table S6), and significant day-by-species interactions for days 1-3 (Additional file 1: Table S10). For each pairwise DE test, we corrected for multiple testing with the Benjamini and Hochberg false discovery rate \cite{RN1372} and genes with an FDR-adjusted P values \textless 0.05 were considered DE unless otherwise stated in the text. 
To find the number of shared DE genes in consecutive time points in each species (Figure 4), we used a two P value cutoff system. To be ``shared? across species for a given pair of time points (e.g. day 0 to 1), a gene must have an FDR-adjusted P value <0.01 in one species and an FDR-adjusted P-value \textless 0.05 in the other species \cite{RN1372}. To estimate the percentage of DE genes in chimpanzees given the observation in humans, we divided the number of genes with an FDR-adjusted P value \textless 0.01 in chimpanzees over the number of genes with an FDR-adjusted P value \textless 0.05 in humans. 

\subsection{Combining technical replicates}

Some analyses did not allow us to model technical replicates explicitly, and treating them as biological replicates would introduce bias in the data. Therefore, we combined technical replicates for the same individual, when available. We calculated the average of the normalized log2(CPM) values for each cell line at each time point. For day 0, 1 human cell line had a pair of technical replicates that were averaged together. For days 1-3, 2 human cell lines had technical replicates that were averaged. We were able to average technical replicates for each of the four chimpanzee cell lines at each time point. After this process, 6 human data points and 4 chimpanzee data points per day remained, for a total of 40 data points. 
When we performed principal components analysis (PCA) using these 40 data points, the results were similar to the PCA plot including all the technical replicates (Figure S9A)-- PC1 was still correlated with day and PC2 was correlated with species (Additional file 1: Table S4E). We visually inspected the PCA plot for the distinct clustering of data points with averaged technical replicates and single replicates in the humans, and this potential pattern was not present, increasing our confidence that this process did not introduce bias into the data. 
We found that the expression values for the 40 samples were robust with respect to the method used to combine the technical replicates. The post-normalization method described above was strongly correlated with a pre-normalization method to combine technical replicates (r \textgreater 0.99 for the 10,304 genes included in the main analysis; Figure S9B). In our pre-normalization method of combining the technical replicates, we summed the raw counts of technical replicates at each time point (for a total 40 data points) and performed the normalization steps described in the ?Transformation and normalization of RNA-sequencing reads? section. 

\subsection{Joint Bayesian analysis with Cormotif}

To cluster genes by their temporal gene expression patterns, we used the R/Bioconductor package Cormotif (version 1.22.0), a method that jointly models multiple pairwise differential expression tests \cite{RN1908}. Unlike other available methods in this class, the Cormotif framework allows for data-set specific differential expression patterns. To identify patterns in expression over time (called ``correlation motifs?), expression levels from days 1-3 were compared to those to the previous day for each gene in each species. Since the program does not allow for the explicit modeling of technical replicates (unlike the voom and limma method above), we first ran the program with the expression values averaged across technical replicates. For more information on this process, see the Methods section on ``Combining technical replicates?.
To use Cormotif, we were required to specify the number of correlation motifs to model. We determined a reasonable range by investigating both the Bayesian information criterion (BIC) and Akaike information criterion (AIC). We observed that the BIC and AIC were minimized across many seeds when 7 or 8 correlation motifs were modeled, respectively (Additional file 1: Table S15; Figure S8A). Thus we further explored models with 7 and 8 correlation motifs. Because Cormotif is not deterministic, we ran Cormotif 100 times and recorded the seed that produced the model with the largest log likelihood (Additional file 1: Table S15). The best model (the seed with the greatest log likelihood) with 7 correlation motifs is displayed in Figure S8B, and the best model with 8 correlation motifs is featured in Figure 5. We selected the model with 8 correlation motifs to be the primary figure because it had a large log likelihood and all motifs contained more than 100 genes. It should be noted, however, that the two models had very similar correlation motifs (expression patterns). 
We were initially conservative when assigning a gene to a specific correlation motif. Following the advice of the Cormotif authors \cite{RN1908}, a gene must have a posterior likelihood estimate of  0.5 to be called DE between time points and < 0.5 to be considered not DE (Additional file 1: Table S9A). We also used this assignment criteria when using Cormotif to compare expression levels using different combination methods (Figure S8C) and to compare all time points to day 0 (Figure S8D). For a trajectory to be defined as DE, the trajectory in humans and chimpanzees needed similar posterior probabilities of differential expression (? 0.20) at each comparison along the trajectory. 
Using topGO \cite{RN1783}, we tested for enrichment of Gene Ontology (GO) biological processes enrichment analysis on various combinations of correlation motifs (Additional file 1: Tables S9B-D). To test for significance, we used the same parameters as \cite{RN1433}. This included the use of Fisher?s Exact Test, with topGO?s weight01 algorithm to account for the correlation among GO categories in its graph structure. Categories with P value \textless 0.01 were considered significant. To determine the categories enriched in Motif 4+7 group only, we ensured that these categories were not significant in any other group (P \textless 0.01; Additional file 1: Tables S9B-D). To test this in a more rigorous way, we then compared the enriched categories for Motif 4 and 7 group given the genes in the two other groups (Additional file 1: Table S9E) using the compareCluster function in the R package clusterProfiler \cite{RN1794}. We used a q value cutoff of 0.05 \cite{RN1372} and set the size of genes annotated by Ontology term for testing parameter between 3 and 3000.

\subsection{Global analysis of variation in gene expression levels} 

We calculated the variance in gene expression level for each gene in each species. Since the largest theoretical range of a variance is from 0 to infinity, we performed a log2 transformation to each variance value (Figure 6). For the analysis with LCLs and the 4 tissues, we used normalized gene expression data from the GTEx Portal (release V6p; gene expression data is in RPKM) from lymphoblastoid cell lines and 4 additional tissues \cite{RN1945, RN2196}. Three of the four tissues are derived from the endoderm germ layer, liver, lung, and pancreas. The heart tissue is mesoderm-derived, for comparison. We identified all of the genes that were expressed and passed GTEx?s filtering criteria (n = 17,542 genes) as well as the individuals that had contributed samples to all 5 tissues (n = 6). 
We then calculated and plotted the log2 variance of the normalized gene expression using the same pipeline as before (Figure S10B). For samples with associated purity values (n = 30), we calculated the effect of purity gene expression levels. We regressed out effect of purity on gene expression levels on a gene-by-gene basis using the linear model function in R in each species independently. We then calculated the log2 variance of these residuals for each gene. We shifted the log2 variances for these residuals so that the median at day 0 for each species = 1. For samples without purity values (n = 32), we calculated the log2 variance of the gene expression levels for each gene, and scaled the values so that the median at day 0 for each species = 1. For each gene at each day for each species, we averaged the scaled log2 variances of the residuals and the log2 variances of the gene expression levels (Figure S10B). We then performed a one-sided t-test between the distribution of log2(variances in gene expression) from day 0 and day 1 in each species, with the alternative hypothesis that the variation was greater in day 0 than day 1. 
We compared the effect sizes of interspecies DE genes with a one-sided Mann-Whitney U test on magnitudes of effect sizes (Figure S10C). We tested the null that there was no change in log2 fold change in gene expression across the species from day 0 to day 1, with the alternative hypothesis that the average magnitude of effect of DE genes (FDR = 5\%) was greater in day 0 than day 1. 

\subsection{Gene-by-gene analysis of variation in gene expression levels and calculating the proportion of true positives}

To determine if there was an enrichment of genes undergoing changes in variation one species, we used an F test to compare two variances in R (var.test command) for each gene using the averaged log2(CPM) expression values of technical replicates. In these tests, the null hypothesis was no change of variance in the gene expression levels between days and the alternative hypothesis was a reduction in variation of gene expression levels between two time points (a one-sided test). 
We calculated the P values for the F statistics from each test and plotted the densities using ggplot2 \cite{RN1389}. If a P value distribution appeared to be even slightly skewed towards small P values, we used the R package qvalue to determine o, the true proportion of null statistics from a given P value distribution \cite{RN1393}. Its complement, 1, is considered the proportion of significant tests from a P value distribution. We used this process to analyze the reduction in variation in each species from days 0 to 1, 1 to 2, 2 to 3, and 0 to 2 (Figures 7A-B).  

Afterwards, we used the same procedure (F tests) to test the alternative hypothesis that the variation of gene expression increased between two time points. We determined o and 1 in the same manner as above to analyze the increase in variation in each species from days 0 to 1, 1 to 2, 2 to 3, and 0 to 2 (Figures 7A-B).  

\subsection{Estimating the proportion of genes that undergo a change in variation in both species}

We then estimated the true proportion of significant genes shared across species for a given set of time points. Rather than take the intersection of the significant genes (for which we would be underpowered), we adopted a method from Storey and Tibshirani 2003 \cite{RN1393}.  This method was recently implemented by Banovich et al. 2016 to determine the sharing of quantitative trait loci (QTLs) from different cell types \cite{RN1399, RN1393}. Using the P value distributions generated in the previous section, we subset the genes in species 2 conditioned on its F statistic significance in species 1 (unadjusted P value \textless 0.05). To test for an enrichment of small P values, we used the P values from species 2 to determine o using the same process as the previous section (Figures 7C-F, Figure S13). We then repeated this process for other P value cutoffs, including 0.01 and 0.10 (Figures S15-S16). To determine robustness with respect to the number of genes considered significant in species 1, we calculated o  for species 2 conditioned on 100 genes with the lowest P values in species 1. This process was repeated for the top 101 to all 10,304 genes (Figure S14). 

\subsection{Estimating the null hypothesis for the proportion of genes that undergo a change in variation in both species}

To determine the null hypothesis for the 1 based on conditioning, we performed permutation tests. First, we combined the unadjusted P values from the F test for a reduction in variation from days 0 to 1 in chimpanzees (species 1) and humans (species 2). We used the randomizeMatrix function in the R package picante \cite{RN1382} to permute the P values of species 1 and then merged this P value distribution with the P value distribution from species 2. We then determined o in species 1 conditioned on its P value significance in species 2 (unadjusted P value \textless 0.05). We repeated this process a total of 100,000 times and found the complement of the 100,000 values (Additional file 1: Table S14). We defined the permuted null hypothesis as the mean 1 value. We then repeated this process, with humans as species 1 and chimpanzees as species 2.
For the enrichment analysis, we classified the genes with a reduced variation of gene expression in both species using a 2 P value cutoff in the results of the F tests (P \textless 0.05 in one species and P \textless 0.10 in the other species). We chose this method because names of the overlapping genes can not be determined by Storey?s approach \cite{RN1393}. As described in the Methods section, ``Joint Bayesian Analysis with Cormotif?, we performed the enrichment analysis using topGO with a significance of P \textless 0.01 \cite{RN1783}. To determine the genes associated with embryonic lethality response to perturbation, we entered our list of genes into the Mouse Genome Database at the Mouse Genome Informatics website provided by the Jackson Laboratory (http://www.informatics.jax.org/batch, \cite{RN1467, RN1446, RN2197, RN1797}) and selected the ``Mammalian Phenotype? option \cite{RN2198}. With this output \cite{RN2198}, we eliminated any feature type that was not a protein coding gene and those genes without an associated phenotype. For each gene list, we calculated how many genes contained at least one of the following terms: ``embryonic lethality?, ``prenatal lethality? or ``lethality throughout fetal growth and development.?

\subsection{Ethics approval and consent to participate}

Human fibroblasts samples for generation of iPSC lines were collected under University of Chicago IRB protocol 11-0524.  Informed consent to participate in the study was obtained from all human participants. The original chimpanzee fibroblast samples for generation of iPSC lines were obtained from Yerkes Primate Research Center of Emory University under protocol 006?12, in full accordance with IACUC protocols \cite{RN1396}. All experimental methods comply with the Helsinki Declaration. 

\subsection{Data availability}

The datasets generated during the current study are available in NCBI's Gene Expression Omnibus \cite{RN1390} repository (GEO Series accession number GSE98411), https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98411.
Source datasets and code can be found at Zenodo \cite{RN2195}. These are available under a Creative Commons Attribution 4.0 International License. All of our analyses using R and the subsequent results can be viewed at https://lauren-blake.github.io/Endoderm\_TC/analysis/.
The GTEx Vp6 dataset is available at the GTEx portal, https://www.gtexportal.org/home/datasets \cite{RN1945, RN2196}. Data regarding genes associated with embryonic lethality response to perturbation is provided by the Mouse Genome Database at the Mouse Genome Informatics website, http://www.informatics.jax.org/batch \cite{RN1797} and our output is provided in \cite{RN2198}.

\subsection{Funding}

LEB was supported by the National Science Foundation Graduate Research Fellowship (DGE-1144082) and by the Genetics and Gene Regulation Training Grant (T32 GM07197). BJP was supported by the Training in Emerging Multidisciplinary Approaches to Mental Health and Disease (T32MH020065). This work was funded by an NIH grant from NIGMS (MH084703). The content presented in this article is solely the responsibility of the authors and the funding bodies had no role in the design of the study and collection, analysis, interpretation of data, and in writing the manuscript.

\section{Acknowledgments}

We thank members of the Gilad, Stephens, and Pritchard labs for helpful discussions and comments on the manuscript. We also thank Irene Gallego Romero for her comments on the manuscript.

The Genotype-Tissue Expression (GTEx) Project was supported by the Common Fund of the Office of the Director of the National Institutes of Health, and by NCI, NHGRI, NHLBI, NIDA, NIMH, and NINDS.

\subsection{Author contributions}

YG and BJP conceived of the study and designed the experiments. SMT and BJP performed the experiments. MM and CC extracted the RNA and prepared the sequencing libraries. LEB, SMT, and BJP analyzed the results (with input from JDB and CJH). YG and BJP supervised the project. LEB, ST, YG, and BJP wrote the paper. All authors read and approved the final manuscript.

%\clearpage\newpage
\section{Supplementary Information}\label{ch03-supplementary-information}

\subsection{Supplementary Methods}

\textit{A correlation-based method designed for short time-series gene expression studies}

In order to classify gene expression trajectories across species, we used a Java program specifically designed for time course microarray data with a small number of time points called Short Time-series Expression Miner (STEM)\cite{RN1363, RN85}. We used TMM-normalized log2(CPM) expression values averaged across technical replicates (n = 40) because it was unclear how we could both model expression with replicates and still use the program to complete analyses related to the overlapping trajectories across species. We note that the main paper used TMM- and cyclic loess- normalized log2 CPM data, but the two datasets are highly correlated (r \textgreater 0.99).
Per the STEM manual?s suggestion for removing gene expression trajectories driven solely by technical noise, we prefiltered our set of 10,304 orthologous genes using a minimum difference of at least one log2 fold change between the minimum and maximum values (not necessarily across consecutive days). We also filtered out the genes that were in the bottom quantile of the within-species correlations. We chose a quantile cutoff rather than a value cutoff for each species because the chimpanzee samples were generally more highly correlated than the human samples. To ensure that the same genes were used for analysis, filtering was done outside of STEM using a custom R script and the union of the gene passing filtering criteria in either species was utilized (i.e., a gene only needed to pass inclusion criteria in one of the two species to be considered for further analysis). 3940 genes remained post-filtering and were used to first run the chimpanzee and human expression data separately (1 run each).
Using the Clustering Method option, we used the ``STEM Clustering Method? and selected the options Maximum Number of Model Profiles = 125 and Maximum Unit Change in Model Profiles between the Time Points = 2.  The former was selected so that we could still maintain a reasonable number of profiles (maximum of 125) and the number of significant model profiles was relatively robust to the maximums unit change in model profiles between time points. 
Since we had previously filtered the data, when we reached the filtering stage in the program, we selected values in the STEM program that would not cause any new genes to be removed. Specifically, we used the following options: Maximum Number of Missing Values = 0, Minimum Expression Change = 0, Minimum Correlation between Repeats = negative 1. We also normalized the data so that the first time point had a gene expression level of 0 and all subsequent values would be log2 fold changes relative to the first time point. 
We used the STEM package to calculate the correlation of each gene to each model profile and to assign each gene to the model profile with which it had the highest correlation. We noticed during early runs of this program using expression values from either species that there were many highly correlated model profiles which we felt would still be well represented but be more interpretable if they were combined into a single profile. Therefore, we selected the option to collapse highly correlated model profiles into a single representative model profile. After each of our 3260 genes were assigned to model profiles, we used the package to determine the significance of the number of genes assigned to each profile by a permutation test. We therefore set options Maximum Correlation = 0.9, Number of Candidate Model Profiles = 1,000,000, Number of Permutations per Gene = All permutations and Significance Level = 0.05 with Correction Method = Bonferroni for the assignment and test of significance steps. 
Through this process, we acquired the lists of all model profiles with at least 1 gene assigned to it, the gene names assigned to each model profile, and which model profiles were significant in each species. We used the ``overlap? option in STEM to determine which profiles were significant in both species or only significant in one species. To find genes that were significantly enriched in correlated clusters, the compare option was used within STEM using a maximum uncorrected intersection P value of 0.005 and only considering profile intersections of at least 20 genes.

\subsection{Supplementary Text}

\textit{Joint Bayesian analysis provides relatively consistent estimates of the conservation of gene expression patterns}

As stated in the main text, the joint modeling technique Cormotif \cite{RN1908} leveraged expression information shared across time points to identify the most common temporal expression patterns. In the main paper, we estimated the degree of conservation of trajectories based on a model with 8 correlation motifs to be 75\%. When we choose the most likely model with 7 correlation motifs instead of 8 and used the same process to calculate the estimate as in the main text, this percentage rose slightly to 80\% (Figure S8B). 
We also calculated the degree of conservation of gene expression trajectories using all 10,304 genes. We did so by using other values generated by Cormotif, the posterior probabilities of differential expression (DE) for every gene at each state transition in each species (e.g. the posterior probability of a given gene being DE in human samples between days 0 and 1). We calculated the percentage of genes with similar posterior probabilities of differential expression (? 0.20) across species at each comparison along the trajectory (e.g. a difference of ? 0.20 between the posterior probabilities of DE in the human and chimpanzee samples from days 0 to 1, as well as from day 1 to 2 and from day 2 to 3). The results suggested relatively strong conservation of gene expression trajectories across species, as 67\% of the 10,304 genes tested had similar posterior probabilities of DE across species at all three comparisons. This percentage was similar (71\%) when we calculated this estimate with 7 correlation motifs instead of 8. Overall, our estimates of the degree of conservation using analysis from Cormotif were relatively consistent (range: 67\%-80\%). 


\textit{The estimates of the conservation of gene expression patterns are relatively consistent when using a joint Bayesian method and a correlation-based method}

The joint modeling approach Cormotif allowed us to utilize the multiple data point structure of our data. There are also some limitations to this method, however. For example, genes are grouped based on the probability that they are DE but the approach does not take into account directionality. Therefore, genes with increased and decreased expression could theoretically be included in the same correlation motif. Furthermore, Cormotif ultimately relies on pairwise comparisons, albeit among all data points. To address some of these considerations, and to ensure that our results are robust we used an additional approach to analyze the data. 
We utilized an approach specifically designed to group genes with similar expression patterns for time-series studies with 8 or fewer time points, called Short Time-series Expression Miner (STEM) \cite{RN1363, RN85}. After filtering genes that only showed small changes in expression as recommended by the program (see Supplementary Methods), 3940 genes with relatively large changes in gene expression levels over the timecourse remained for analysis. In the human samples, 3260 of these genes clustered into 9 significant model profiles (Figure S7B). (There were an additional 20 profiles that did not contain enough genes to reach significance.) 2846 were clustered into 6 significant model profiles in the chimpanzees (Figure S7B). (There were 24 additional non-significant profiles for the chimpanzee samples.) All of the significant chimpanzee profiles are also significant profiles using human samples, highlighting the similarities of gene expression trajectories across the species. Supporting our assessment of the high degree of conservation, the 3 significant human profiles that did not reach significance using chimpanzee samples nevertheless contained the same ranking (by number of genes) in each species. 
To determine the proportion of genes with conserved trajectories for this method, we analyzed all genes assigned to the same cluster (1685 genes). We then identified an additional 1020 genes in similar clusters (mean profile correlation = 0.73, see Supplementary methods below).  Using this approach, we discovered 2705 genes (or 69\% of the 3940 genes analyzed) with the same or a highly correlated trajectory across species. This high degree of conservation across the timecourse is very similar to the estimates provided by the joint Bayesian method. 
We found that the STEM clustering frequently assigned known regulators or markers of the differentiation process to the same cluster. This increased our confidence in the results, as we had a prior expectation that these markers would be assigned to the same or similar profiles for both species. For example, \textit{EOMES} and \textit{MIXL1}, transcription factors essential for endoderm formation, are assigned to the same clusters across species (Figure S7A). Furthermore, of all the developmental regulators and markers examined (a total of 34 gathered from literature) the majority (70\%) are assigned to the same cluster in both species, and an additional 20\% assigned to highly similar profiles (correlation = 0.70). Most of these 34 genes (61\%) are assigned to just two profiles, suggesting that these profiles are likely the representative trajectories for developmental progression and drivers of endoderm specification.


\subsection{Supplementary Figures}


\subsection{Supplementary Tables}

\begin{table}[!htb]
\caption[S1. Purity estimates for each of the 63 samples.]{\textbf{S1. Purity estimates for each of the 63 samples.} Column 1 contains the Sample ID and Day. Column 2 contains the purity estimate. For more information about how purity was estimated, see Table S4b. Samples without purity estimates were assigned an ``NA'' designation. }
\label{tab:ch02-s1}
\end{table}

\begin{table}[!htb]
\caption[S2. RNA Integrity Number (RIN) measured with a Bioanalyzer (Agilent) for each of the 63 samples.]{\textbf{S2. RNA Integrity Number (RIN) measured with a Bioanalyzer (Agilent) for each of the 63 samples.} Column 1 contains the Sample ID and Day. Column 2 contains the RIN score.  Samples without RIN scores are annotated with an ``NA'' designation.}
\label{tab:ch02-s2}
\end{table}


\begin{table}[!htb]
\caption[S3. TMM- and cyclic loess-normalized log2 counts per million for the 10,304 genes analyzed in this study for each of the 63 samples.]{\textbf{S3. TMM- and cyclic loess-normalized log2 counts per million for the 10,304 genes analyzed in this study for each of the 63 samples.} Column 1 provides the Ensembl gene name. Row 1 contains the Sample ID and Day (to the left and right of the hyphen, respectively).} 
\label{tab:ch02-s3}
\end{table}

\begin{table}[!htb]
\caption[S4. Biological and technical factors of interest, and the accompanying values for each of the 63 samples.]{\textbf{S4. Biological and technical factors of interest, and the accompanying values for each of the 63 samples.} (A) This file contains sample information (including identifiers and demographics), experimental details, nucleic acid extraction information, library preparation information, sequencing information, and quality metrics. Row 1 contains the type of information. Row 2 contains a description of the variable, including units of measurement, if applicable. Row 3 lists the variable name. Rows 4-66 provide the values for each variable for the 63 samples. Row 67 contains information for the outlier sample, H1B. (B) A glossary of the technical factors and their definitions for this study.}  
\label{tab:ch02-s4}
\end{table}

\begin{table}[!htb]
\caption[S5. Information about the sharing of DE genes across species for a given transition (e.g. day 0 to 1).]{\textbf{S5. Information about the sharing of DE genes across species for a given transition (e.g. day 0 to 1).} Column 1 lists the days which were tested for DE genes in each species. The ``pval'' refers to the Bonferroni-Hochberg adjusted p-value.}
\label{tab:ch02-s5}
\end{table}

\begin{table}[!htb]
\caption[S6. Gene Ontology results.]{\textbf{S6. Gene Ontology results.} GO when comparing the two most divergent expression patterns, motifs 4 and 7 from Figure 5, to the 10,304 genes studied. The output is from the Panther database testing for an overrepresentation of genes in given GO biological process categories, using the Bonferroni correction for multiple testing.}
\label{tab:ch02-s6}
\end{table}

\begin{table}[!htb]
\caption[S7. The number of interspecies DE genes under various conditions.]{\textbf{S7. The number of interspecies DE genes under various conditions.} Column 1 lists the number of samples and FDR cutoff. All tests used TMM- and cyclic loess-normalized log2(CPM) unless otherwise stated. Columns 2-10 list the total number of DE genes and day-specific DE genes at each day. The same voom+limma pipeline was used to generate all DE statistics (see Methods, Table 2, Figure 3a).}
\label{tab:ch02-s7}
\end{table}

\begin{table}[!htb]
\caption[S8. P-values from model comparisons for each recorded technical variable.]{\textbf{S8. P-values from model comparisons for each recorded technical variable.} Column 1 lists the technical variables. The technical factor type in Column 2 corresponds to the data type (1 = categorical variable with more than two levels, 2 = categorical variable with two levels, 3 = numerical variable), which impacted the model type (Methods). P-values were generated for models of the technical factors from days 0-1 (Column 3) and from days 0-3 (Column 4). Non-significant P-values in all of the day 0-1 tests and most of the day 0-3 tests suggests that variation was relatively well distributed between the days and between the species.} 
\label{tab:ch02-s8}
\end{table}

\begin{table}[!htb]
\caption[S9. Distributions of P values from all F tests.]{\textbf{S9. Distributions of P values from all F tests. } (A) P-value distributions from F tests of reduced variation from day 0 to 1, 1 to 2, 2 to 3, 1 to 3 in each species. (B) P-value distributions from F tests of increased variation from day 0 to 1, 1 to 2, 2 to 3, 1 to 3 in each species.}
\label{tab:ch02-s9}
\end{table}

\begin{table}[!htb]
\caption[S10. 100,000 pi zero values calculated from permuted P-values obtained from F tests for reduced variation in gene expression levels.]{\textbf{S10. 100,000 pi zero values calculated from permuted P-values obtained from F tests for reduced variation in gene expression levels.}  Column 1 contains 0 estimates for the chimpanzee samples conditioned on significance in human samples based on permutations (mean = 0.274, standard deviation = 0.046). Column 2 is the same data type but was calculated using human samples conditioned on significance in chimpanzees (mean = 0.825, standard deviation = 0.050).}
\label{tab:ch02-s10}
\end{table}


\begin{table}[!htb]
\caption[S11. Log-likelihoods for Cormotif models using 7 and 8 correlation motifs.]{\textbf{S11. Log-likelihoods for Cormotif models using 7 and 8 correlation motifs.} Columns 1 and 2, respectively; seeds 1-100 were used to generate both lists.}
\label{tab:ch02-s11}
\end{table}

