\chapter{Conclusion}\label{conclusion}

In Chapter \ref{ch:tb}, I described my work investigating the innate
immune response to \emph{Mycobacterium tuberculosis} (MTB)
\citep{Blischak2015}. Previous studies had identified the genes which
are differentially expressed upon infection with MTB, and some have
even compared the differences between the reponse to strains of MTB
that vary in their virulence. The first novelty of our study was to
include other bacteria in the infection experiments. Specifically, we
included the following Mycobacteria: two strains of virulent MTB,
avirulent (heat-inavtivated) MTB, the attentuated \emph{Mycobacterium
  bovis} (used as a vaccine), and the avirulent \emph{Mycobacterium
  smegmatis}. The non-mycobacteria species we included were
\emph{Yersinia pseudotuberculosis}, \emph{Salmonella typhimurium}, and
\emph{Staphylococcus epidermidis}. This allowed us to distinguish
between the innate immune response to MTB versus other virulent
bacteria, MTB versus avirulent mycobacteria, amd MTB versus deceased
MTB.

This novel study design comparing many bacterial infections to isolate
the innate immune respone to MTB also posed analytical
challenges. Standard differential expression analyses (or in general
any large scale testing of thousands or more genomic features) are
well-suited for experiments with a few conditions. For example, the
most common approach is to perform pairwise differential expression
tests and then overlap the lists of differentially expressed
genes. These results are always biased by incomplete power. Because
hypothesis testing uses an arbitrary p-value threshold to determine
statistical significance, a gene with a p-value below this threshold
for one comparison but a p-value slightly above this threshold for a
separate comparison will be classified as specific to the first when
in reality the gene is behaving similarly in both. As the number of
pairwise comparisons increases, the problem of incomplete power is
exacerbated, i.e. a gene is more likely to be statistically
significant for some susbset of comparisons. This increase in
comparisons also decreases the ability to interpret the results. A
3-way Venn diagram (and perhaps a 4- or 5-way) can be interpreted, but
this approach breaks down with additional comparisons.

It is known that the innate immune response is important for fighting
MTB infections. Alveolar macrophages are the primary target of MTB,
and they initiate the formation of granulomas to sequester
MTB. Furthermore, vaccines against TB have had limited efficacy. To
identify human genes which are important for the response to MTB
infection, we isolated macrophages from 6 healthy donors and infected
them with MTB, avirulent mycobacteria, and virulent
non-mycobacteria. To account for incomplete power that would lead to
false positive MTB-specific findings, I jointly analyzed the gene
expression response data using a Bayesian framework implemented in the
R/Bioconductor software package, Cormotif. Using this method, I
identifed hundreds of genes that respond specifically to infection
with MTB and closely related mycobacterial species. These genes are
candidates for containing genetic variants which affect TB
susceptibility. This study was published last year
\citep{Blischak2015}.

In Chapter \ref{ch:tb-suscept}...

To investigate how the innate immune cells of susceptible individuals
function compared to those of resistant individuals, we collected
primary dendritic cells from individuals that had recovered from TB
(i.e.~susceptible) and individuals that tested positive for latent MTB
infections but had not developed TB (i.e.~putatively resistant). The
dendritic cells were infected with MTB, and we performed RNA-seq on
the infected and non-infected cells. I have identified hundreds of
genes which are differentially expressed between susceptible and
resistant individuals. Furthermore, I built a classifier to
differentiate between suceptible and resistant individuals based on
their gene expression profiles.

In Chapter \ref{ch:singleCellSeq}...

The previous studies were performed on bulk populations of cells,
which lacked the resolution to observe the cell-to-cell heterogeneity
in the immune response. Single cell RNA-sequencing (scRNA-seq)
technology can capture the gene expression profile of individual
cells. However, the technology is still relatively new, and best
practices for study design have yet to be established. In this study,
we collected 3 replicates of induced pluripotent stem cells (iPSCs)
from 3 Yoruba individuals from the HapMap Project. We determined the
minimum number of cells and sequencing depth required to resemble the
bulk population and detect most expressed genes, identified the
technical effects introduced by the batch processing, and developed a
pre-processing pipeline to reduce the technical variation. Lastly, we
provide a proof of concept for an efficient design for future single
cell studies. By identifying the origin of a cell using the
polymorphisms present in the RNA-seq reads, we could include cells
from multiple individuals in each processing batch. This allows for
accounting of technical batch effects while minimizing the total
amount of replication. This study was recently posted as a pre-print
on bioRxiv and submitted for review \citep{Tung2016}.

I have identified hundreds of genes involved in fighting MTB
infections.  More broadly, I have demonstrated that a joint Bayesian
model is an effective tool for analyzing the large-scale multivariate
data produced by genomic studies. Lastly, I have determined an
effective study design for future single cell studies.
